invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -16,6 +16,12 @@
         "abstracts": {
           "type": "keyword"
         },
+        "document_types": {
```

Comment:
Shouldn't be singular this one?

Reply:
I created another ticket for deciding if in the schemas we should name arrays as plural or singular. For now I will keep it like this, because all other arrays in the are plural and I would like to keep it consistent.